crypto/secp256r1/verifier.go

```diff
@@ -0,0 +1,22 @@
+package secp256r1
+
+import (
+	"crypto/ecdsa"
+	"math/big"
+)
+
+// Verify verifies the given signature (r, s) for the given hash and public key (x, y).
+// It returns true if the signature is valid, false otherwise.
+func Verify(hash []byte, r, s, x, y *big.Int) bool {
+	// Create the public key format
+	publicKey := newPublicKey(x, y)
+
+	// Check if they are invalid public key coordinates
+	if publicKey == nil {
```

Comment:
I feel like for `newXXX` we will expect a obj created (instead of nil pointer returned) if not error is reported.

I prefer 
```
func newPublicKey(x, y *big.Int) (*ecdsa.PublicKey, error)
...


```

Reply:
Your suggestion makes sense, but I think for this feature it's more important to maintain consistency with other implementations.